RELEASE_NOTES.md

```diff
@@ -2,85 +2,58 @@
+The `microgrid` package now exposes grid connections uniformly and introduces formula operators for `consumption` and `production`, replacing the `logical_meter.*_{production,consumption}()` formulas. The `actor` package restarts crashed actors with a delay, and the `ConnectionManager` exposes the `microgrid_id` and `location` details.
+
+There are also a few bug fixes, documentation improvements and other minor breaking changes.
 
 ## Upgrading
 
-- `microgrid.battery_pool()` method now accepts a priority value.
+- `actor` package
 
-- `microgrid.grid()`
+  * Actors are now restarted after a small delay when they crash to avoid a busy loop and spamming the logs if the actor keeps failing to start.
 
-  * Similar to `microgrid.battery_pool()`, the Grid is now similarily accessed.
+- `microgrid` package
 
-- `BatteryPool`'s control methods
+  * `grid`: The grid connection is now exposed as `microgrid.grid()`. This is more consistent with other objects exposed in the `microgrid` module, such as `microgrid.battery_pool()` and `microgrid.logical_meter()`.
 
-  * Original methods `{set_power/charge/discharge}` are now replaced by `propose_{power/charge/discharge}`
-  * The `propose_*` methods send power proposals to the `PowerManagingActor`, where it can be overridden by proposals from other actors.
-  * They no longer have the `adjust_power` flag, because the `PowerManagingActor` will always adjust power to fit within the available bounds.
-  * They no longer have a `include_broken_batteries` parameter.  The feature has been removed.
+  * `BatteryPool`: The `include_broken_batteries` argument was removed from the control methods. This option is no longer supported.
 
-- `BatteryPool`'s reporting methods
+  * `ComponentGraph`: The component graph is now exposed as `microgrid.component_graph.ComponentGraph`.
 
-  * `power_bounds` is replaced by `power_status`
-  * The `power_status` method streams objects containing:
-    + bounds adjusted to the actor's priorities
-    + the latest target power for the set of batteries
-    + the results from the power distributor for the last request
+  * `logical_meter()`: The `*_consumption()` and `*_production()` methods were removed. You should use the new `consumption` and `production` formula operators instead.
 
-- Move `microgrid.ComponentGraph` class to `microgrid.component_graph.ComponentGraph`, exposing only the high level interface functions through the `microgrid` package.
+    For example:
 
-- An actor that is crashing will no longer instantly restart but induce an artificial delay to avoid potential spam-restarting.
+    ```python
+    # Old:
+    pv_consumption = logical_meter.pv_consumption_power()
+    production = (logical_meter.pv_production_power() + logical_meter.chp_production_power()).build()
+    # New:
+    pv_consumption = logical_meter.pv_power().consumption().build()
+    production = (logical_meter.pv_power().production() + logical_meter.chp_power().production()).build()
```

Comment:
Can someone double-check this example is OK? @matthias-wende-frequenz maybe?

Reply:
this looks correct.